operations/README.md

```diff
@@ -82,6 +82,15 @@ This section describes all operations supported by the API, organised here by th
 | [Add alternative payment](payments.md#add-alternative-payment) | Adds a new alternative payment to a specified customer |
 | [Add outlet bills](outletbills.md#add-outlet-bills) | Adds new outlet bills with their items |
 
+## Loyalty memberships
```

Comment:
I think we should put both sets of operations, for **loyalty memberships** and **loyalty programs**, under the same heading 'Loyalty' on this page, as these headings are meant to represent general themes like 'reservations' or 'configuration' rather than specific domain entities. Thank you.

Reply:
Done 😊